404.html

```diff
@@ -1,6 +1,13 @@
 ---
 ---
-
+  <script type="text/javascript">
```

Comment:
@ljacomet I reduced the scope of redirection by adding ?redirect=true, for example, http://ehcache.org/documentation/3.1/expiry.html?redirect=true will redirect you to doc root, but without the param, it will show you 404.html